### PR TITLE
(RE-8952) Create puppet5 release packages

### PR DESCRIPTION
**Pull request by melissa**
This commit allows us to build the puppet5-release packages. The gpg
keys delivered are only the main signing key, and not the nightly
signing key.

### CODE REVIEW

**Comment by melissa** [comment]
This only delivers the main signing key, not any of our other keys

File: configs/projects/puppet5-release.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+project 'puppet5-release' do |proj|
+  proj.description 'Release packages for the Puppet5 repository'
+  proj.release '1'
+  proj.license 'ASL 2.0'
+  proj.version '5.0.0'
+  proj.vendor 'Puppet, Inc. <release@puppet.com>'
```
Comment (melissa) [inline configs/projects/puppet5-release.rb]: I know we had an issue with the sles vendor, but as these are brand new packages, I figured it'd be safe to update this here

Comment (shrug) [inline configs/projects/puppet5-release.rb]: I think so. easy to test as well

File: configs/projects/puppet5-release.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  proj.homepage 'https://www.puppet.com'
+  proj.target_repo 'puppet5'
+  proj.noarch
+
+  proj.conflicts 'puppet-release'
+  proj.conflicts 'puppet-nightly-release'
```
Comment (melissa) [inline configs/projects/puppet5-release.rb]: @underscorgan @shrug

**Pull request merged**